schema/iiif_3_0.json

```diff
@@ -142,17 +159,25 @@
                         }
                     ]
                 },
+                "service": { "$ref": "#/classes/service" },
                 "format": { "$ref": "#/types/format" },
-                "height": {
```

Comment:
How come you have removed height and width from resource? I think you need it for static non IIIF images. In particular for thumbnails:
```
"thumbnail": {
      "id": "https://example.org/identifier/thubmnail.jpg",
      "type": "Image",
      "format": "image/jpeg",
      "height": 100,
      "width": 100
}
```